samples/Checksums/single_hash.crc

```diff
@@ -0,0 +1 @@
+ffffffff
```

Comment:
IMO, the `.crc` extension should be removed. Even in hexadecimal, something innocuous and random has a high risk of matching something unrelated:

~~~console
$ ls -1 github.com/random-designer/website-mockup/swatches/solid-colours/*
background.crc
foreground.crc
~~~

Here, "CRC" could stand for an ad-hoc abbreviation of _"Centred Rectangle Colour"_, containing the hex-code for an RGBA (or ARGB) colour:

~~~cson
# solid-black.crc
00000000
~~~

Not saying this is likely, but more that this is the sort of random junk that tends to frequent search-results for a very generic-looking extension. I'm basing that on precedent search efforts... you wouldn't believe the amount of weird and wonderful crap people upload to GitHub. 😉

It'd be different if `.crc` files had a common-enough syntax that we could target with a heuristic, e.g.,

~~~py
CRC32 (filename) = 00000000
~~~

… but I doubt that's commonplace enough (haven't checked).

Reply:
I didn't see BSD-style checksums for `.crc`, but I saw [another format similar to INI files on the second page](https://github.com/search?p=2&q=extension%3Acrc+CRC32):
```
filename=FILE
size=1337
crc32=00000000
```